### DISCUSSION TITLE
[灌水]原画集要开了，各位加油

### DISCUSSION THREAD

**Discussion by 596wjr** [share121/inter-knot#4818] [崩坏3]
众所周知，崩崩崩在7.8开原画集活动，各位老登准备好了吗
![Screenshot_2024-09-06-22-58-39-206_com miHoYo enterprise NGHSoD](https://github.com/user-attachments/assets/245514e6-0979-4cd9-904b-e3e0ea82559f)

**Comment by canyusuo** [comment]
什么7.8，不是7.9？<img src="https://upload-bbs.miyoushe.com/upload/2022/12/08/a2fa805a0a782cdb8ab44ddaf8d5343d_7408497524870727211.png" alt="阿姬倒地" width="75" >

**Comment by flystarnxovo** [comment]
不是，下版本没有原画集，真的<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/e865802e4393f62481883d43990e4d95_6045795287664860169.gif" alt="梅比乌斯-海滨" width="75" >

**Comment by Myitian** [comment]
![Screenshot_2024-09-19-15-02-47-56_46bf1df0679baeb0b0b71d0d22d9057a.jpg](https://github.com/user-attachments/assets/49ce67bd-797f-445e-88dd-0ee5c522ddb5)

（

**Comment by aokamul** [comment]
啊
